### PR TITLE
Update extending a plugin with a plugin article 

### PR DESCRIPTION
**Pull request by mitelg**
remove unnecessary things

### CODE REVIEW

File: exampleplugins/5.2/SwagExtendCustomProducts/Decorators/FileTypeWhiteListDecorator.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+class FileTypeWhiteListDecorator implements FileTypeWhitelistInterface
+{
+    /**
+     * @var FileTypeWhitelistInterface
+     */
+    private $fileTypeWhitelist;
+
+    /**
+     * inject the original FileTypeWhiteListDecorator
```
Comment (ngoedde) [inline exampleplugins/5.2/SwagExtendCustomProducts/Decorators/FileTypeWhiteListDecorator.php]: inject -> Inject. Always start sentences with an upper case character.

File: exampleplugins/5.2/SwagExtendCustomProducts/Resources/Views/backend/swag_custom_products/view/option/types/custom_type.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+//
```
Comment (ngoedde) [inline exampleplugins/5.2/SwagExtendCustomProducts/Resources/Views/backend/swag_custom_products/view/option/types/custom_type.js]: I don't think that you need this.

File: exampleplugins/5.2/SwagExtendCustomProducts/Resources/Views/frontend/swag_custom_products/options/customType.tpl [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{block name="frontend_detail_swag_custom_products_options_customtype"}
+    <input class="wizard--input" type="text" name="custom-option-id--{$option['id']}"
```
Comment (ngoedde) [inline exampleplugins/5.2/SwagExtendCustomProducts/Resources/Views/frontend/swag_custom_products/options/customType.tpl]: {$option.id}

File: exampleplugins/5.2/SwagExtendCustomProducts/Resources/Views/frontend/swag_custom_products/options/customType.tpl [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+           id="custom-products-option-{$key}"
+           data-field="true"
+        {if $option['required']}
```
Comment (ngoedde) [inline exampleplugins/5.2/SwagExtendCustomProducts/Resources/Views/frontend/swag_custom_products/options/customType.tpl]: $option.required

File: exampleplugins/5.2/SwagExtendCustomProducts/Decorators/FileTypeWhiteListDecorator.php (hunk 1 above)
Comment (mitelg) [reply exampleplugins/5.2/SwagExtendCustomProducts/Decorators/FileTypeWhiteListDecorator.php]: done

File: exampleplugins/5.2/SwagExtendCustomProducts/Resources/Views/backend/swag_custom_products/view/option/types/custom_type.js (hunk 2 above)
Comment (mitelg) [reply exampleplugins/5.2/SwagExtendCustomProducts/Resources/Views/backend/swag_custom_products/view/option/types/custom_type.js]: done

File: exampleplugins/5.2/SwagExtendCustomProducts/Resources/Views/frontend/swag_custom_products/options/customType.tpl (hunk 3 above)
Comment (mitelg) [reply exampleplugins/5.2/SwagExtendCustomProducts/Resources/Views/frontend/swag_custom_products/options/customType.tpl]: nope. it's an array

File: exampleplugins/5.2/SwagExtendCustomProducts/Resources/Views/frontend/swag_custom_products/options/customType.tpl (hunk 4 above)
Comment (mitelg) [reply exampleplugins/5.2/SwagExtendCustomProducts/Resources/Views/frontend/swag_custom_products/options/customType.tpl]: nope. it's an array

**Pull request merged**